src/screens/AdminPortal/FundCampaignPledge/modal/PledgeModal.module.css

```
@@ -15,29 +15,281 @@
+}
+
+.fieldRow {
+  width: 100%;
+  margin: 0;
+}
+
+.fieldRow :global(label) {
+  display: inline-block;
+  margin-bottom: var(--space-2);
+  color: var(--color-black);
+  font-family: 'Inter', sans-serif;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)